NEWS

```diff
@@ -7,7 +7,7 @@ NEWS for OpenSC -- History of user visible changes
  * Add threading test to `pkcs11-tool` (#2067)
  * Add support to generate generic secret keys (#2140)
  * `opensc-explorer`: Print information about LCS (Life cycle status byte) (#2195)
- * Add support for Apple's arm64 (M1) binaries, removed TokenD. A seperate installer with TokenD (and without arm64 binaries) will be available (#2179).
+ * Add support for Apple's arm64 (M1) binaries, removed TokenD. A separate installer with TokenD (and without arm64 binaries) will be available (#2179).
```

Comment:
I think the `NEWS` file should be append only. I was already turned down with one typo fix in there, but lest see what others will think.

Reply:
OK, I can revert that if needed. I can understand NEWS is a pristine copy of past NEWS, with their typos. Wait and see?